### PR TITLE
Added view in gallery

### LINKED ISSUE #564: Add option to `View images in Gallery` in  PDF to Images and Extract Images
- In  PDF to Images and Extract Images, add a new option (after the images are created) - `View Images in Gallery`
- On clicking this, open the folder in which images are created in the gallery

### PR DESCRIPTION
**Pull request by jddeep**
# Description
Added View in gallery button and functionality.
screenshot_20190303-151634

screenshot_20190303-152049

Fixes #564 

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by jddeep** [comment]
@Swati4star please review and let me know.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -363,6 +363,7 @@
     <string name="select_pdf_file">Select PDF File</string>
     <string name="share_images">Share Images</string>
     <string name="view_images">View Images</string>
+    <string name="view_in_gallery">View Images in Gallery</string>
```
Comment (Swati4star) [inline app/src/main/res/values/strings.xml]: add this string for files of other locales too

File: app/src/main/res/layout/fragment_pdf_to_image.xml [hunk 2]
```diff
@@ -88,7 +111,7 @@
                 android:text="@string/view_images" />
 
         </LinearLayout>
-
+        </LinearLayout>
```
Comment (Swati4star) [inline app/src/main/res/layout/fragment_pdf_to_image.xml]: indent the code

**Comment by jddeep** [comment]
@Swati4star Done. Please review. Thanks.

File: app/src/main/res/layout/fragment_pdf_to_image.xml [hunk 3]
```diff
@@ -58,37 +58,60 @@
             android:layout_height="wrap_content"
             android:layout_alignParentBottom="true"
             android:layout_centerHorizontal="true"
-            android:gravity="center"
+            android:orientation="vertical"
             android:visibility="gone">
 
             <Button
-                android:id="@+id/shareImages"
+                android:id="@+id/viewImagesinGallery"
```
Comment (srv-twry) [inline app/src/main/res/layout/fragment_pdf_to_image.xml]: `i` should be capital.

File: app/src/main/res/layout/fragment_pdf_to_image.xml [hunk 4]
```diff
@@ -58,37 +58,60 @@
...
-                android:text="@string/share_images" />
+                android:padding="15dp"
+                android:text="@string/view_in_gallery"
+                android:textColor="?attr/bottomSheetTextColor" />
 
-            <Button
-                android:id="@+id/viewImages"
-                android:layout_width="wrap_content"
+            <LinearLayout
+
```
Comment (srv-twry) [inline app/src/main/res/layout/fragment_pdf_to_image.xml]: remove extra line.

File: app/src/main/res/layout/fragment_pdf_to_image.xml (hunk 3 above)
Comment (Swati4star) [reply app/src/main/res/layout/fragment_pdf_to_image.xml]: @jddeep Fix these formatting changes too

**Comment by jddeep** [comment]
@Swati4star All changes are done. Thanks.

**Pull request merged**